charts/rstudio-pm/values.yaml

```diff
@@ -85,6 +85,8 @@ service:
   nodePort: false
   # -- The Service port. This is the port your service will run under.
   port: 80
+  # -- The port to forward to on the Package Manager pod. Also see pod.port
+  targetPort: 4242
```

Comment:
Besides `service.targetPort` and `pod.port`, there are a bunch of other `4242` references in the values, like the `config.HTTP.Listen = :4242` setting, liveness/startup probe. I assume users would have to find-and-replace all `4242` references manually to fully change the running port?

Is it possible to use a template variable here so you only have to set `4242` once somewhere?

Otherwise this is fine of course, as it's still better than what we had before.